backend/app/summary.py

```diff
@@ -267,7 +267,7 @@ def fetch_summary_bytes(url: str) -> bytes | None:
 
 
 def update(sqldb) -> None:
-    current_apps = set(apps.get_appids())
+    current_apps = set(apps.get_appids(include_eol=True))
```

Comment:
Should not need it and IMO might be bad, as it can cause a meilisearch entry due to this